crates/config/src/etherscan.rs

```diff
@@ -134,10 +134,12 @@ impl ResolvedEtherscanConfigs {
         self,
         chain: Chain,
     ) -> Option<Result<ResolvedEtherscanConfig, EtherscanConfigError>> {
-        for (_, config) in self.configs.into_iter() {
+        for (name, config) in self.configs.into_iter() {
             match config {
                 Ok(c) if c.chain == Some(chain) => return Some(Ok(c)),
-                Err(e) => return Some(Err(e)),
+                Err(e) => {
+                    warn!("invalid etherscan config for {:?}: {:?}", name, e);
```

Comment:
Hi @andelf thanks for your PR!

Would you mind changing `warn!` to `let _ = sh_warn!(..)`

This will make it so the warning is outputted to the user's terminal rather than to the `tracing` target for internal debugging

Would you mind adding a unit test that validates the change?

Thanks!